package.json

```diff
@@ -46,29 +46,29 @@
     "web-animations"
   ],
   "peerDependencies": {
-    "react": "0.13.x || 0.14.x || 15.x.x",
-    "react-dom": "0.13.x || 0.14.x || 15.x.x"
+    "react": ">0.13.x <16.x.x",
+    "react-dom": ">0.13.x <16.x.x"
   },
   "devDependencies": {
-    "@kadira/storybook": "2.30.1",
-    "@types/react": "15.0.24",
-    "babel": "6.1.18",
-    "babel-cli": "6.2.0",
-    "babel-core": "6.2.1",
+    "@storybook/react": "^3.2.8",
```

Comment:
Nit: can we omit the `^`?

I don't really think this even matters anymore, with Yarn/NPM5. But, my inner OCD wants it to be consistent, haha.

Reply:
You probably also want to upgrade it to 3.2.12, there has been some critical bugfixes